src/CommitmentsV2/SFA.DAS.CommitmentsV2.UnitTests/Application/Queries/CanAccessApprenticeship/CanAccessApprenticeshipQueryHandlerTests.cs

```diff
@@ -120,27 +118,14 @@ public CanAccessApprenticeshipQueryHandlerTestsFixture SeedData(bool isApproved
                 ProviderId = _providerId,
                 Id = _cohortId,
             };
-            if (isApproved)
-            {
-                apprenticeship = new ApprovedApprenticeship
-                {
-                    Id = _apprenticeshipId,
-                    CommitmentId = _cohortId,
-                    Cohort = _cohort,
-                    AgreementStatus = AgreementStatus.BothAgreed,
-                };
-            }
-            else
+            
+            var apprenticeship = new Apprenticeship
             {
-                apprenticeship = new DraftApprenticeship
```

Comment:
@chrisfoster76 From the merge, I assumed that DraftApprenticeship had been removed